README.md

```diff
@@ -37,7 +37,7 @@ Before contributing or adding a new feature, please make sure you have already i
 ## 🛠️ Installation Steps
 
 1. Fork the project
-2. Clone the project
+2. Clone the project by running `git clone https://github.com/EddieHubCommunity/LinkFree.git`
```

Comment:
Now that I think about it, the fork is the repository the user must clone, not the original repository.
```suggestion
2. Clone the project by running `git clone https://github.com/<your-username>/LinkFree.git`
```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inefficient

Reply:
Oh sorry, I reviewed without noticing this!

Reply:
No worries @CodingSpecies, it does happen! We all learn from mistakes! 😄

Reply:
May I know, why you are **marking this conversation resolved** again and again **without making changes**? We'd love to know your Point of view!! 

I think these changes should be made, to be very specific, especially for the first timers!! 
What do you think!?